### ISSUE TITLE
Diacritics are not supported

### ISSUE THREAD

**Issue by filmil** [bdjukic/glasajDijasporo#5]
It seems latin letters with diacritics aren't supported.

**Comment by icemarkom** [comment]
Specifically, letters "ć", "Ć", "Č", and "č" do not appear in the generated PDF. My experience when filling out the form from Chrome 56.0.2924.87 on Linux.

**Comment by bdjukic** [comment]
Yep, confirming that there seems to be a problem with the PDF component that I'm using when it comes to rendering of Latin/Cyrillic characters. In my case Latin characters are fine, but Cyrillic are completely missing. @filmil @icemarkom would guys mind filling out the form on a different device? I'm using iPhone (Safari) for testing and Surface (Edge) for development.

**Comment by icemarkom** [comment]
Interestingly enough, I just used Chrome on a Mac (56.0.2924.87), and it it worked fine here.

**Comment by filmil** [comment]
Content encoding?
